cachelib/cachebench/runner/CacheStressor.h

```diff
@@ -77,7 +77,7 @@ class CacheStressor : public Stressor {
         std::unique_lock<folly::SharedMutex> lock;
 
         CacheStressSyncObj(CacheStressor& s, std::string itemKey)
-            : lock{s.chainedItemTryAcquireUniqueLock(itemKey)} {}
```

Comment:
I think we also need to remove the `chainedItemTryAcquireUniqueLock` implementation because it was added by us, right?

Reply:
yes - forgot that part